doc/en/reference/reference.rst

```diff
@@ -1892,8 +1892,11 @@ All the command-line flags can be obtained by running ``pytest --help``::
                             tests. Optional argument: glob (default: '*').
       --cache-clear         Remove all cache contents at start of test run
       --lfnf={all,none}, --last-failed-no-failures={all,none}
-                            Which tests to run with no previously (known)
-                            failures
+                            With ``--lf``, determines whether to execute tests when there
```

Comment:
I just remembered this text is actually automatically generated, but no harm updating it manually (it will be overwritten later).

Reply:
Ohh I didn't know that actually. 
Good to know, is that why the help code in cacheprovider and in reference are the same?

Reply:
Yes!